AI4PEX_Data_Questionnaire/questionnaire.md

```diff
@@ -0,0 +1,61 @@
+We'd appreciate it if you had 5min to take this survey, this will help us gauge the amount of compute
+resources we'd need to have for the AI4PEX project (both available, and to request new allocations). Very many thanks 
+in advance! :beer:
+
+- 1. What datasets are you planning to use in your work in EXPECT? Choose as many as you like:
+  - CMIP6
+  - LSFMIP
+  - Climate Digital Twin DestinE
+  - EERIE, NextGEMS
+  - Mars seasonal forecasts
+  - ERA5
+  - ESA CCI observations
+  - Simulations generated within AI4PEX
+  - Not sure (yet)
+  - Others (please tell us)
+- 2. What data frequencies do your analysis need? Choose as many as you like:
```

Comment:
Should probably ask for spatial resolution and if area will be global or if not the extent in lat/lon. Ocean data will be very high resolution but not global as I understand.

Reply:
likewise good call! Done in https://github.com/AI4PEX/Housekeeping/pull/3/commits/758321d62accdfdd0e406b777fcd43414251713c